descope/sdk/mgmt.go

```diff
@@ -37,6 +37,13 @@ type Tenant interface {
 
 	// Load all project tenants
 	LoadAll() ([]*descope.Tenant, error)
+
+	// Search all tenants according to given filters
+	//
+	// The options optional parameter allows to fine-tune the search filters
+	// and results. Using nil will result in a filter-less query with a set amount of
+	// results.
+	SearchAll(options *descope.TenantSearchOptions) ([]*descope.Tenant, error)
```

Comment:
just noting the comment about the `searchAll` term here as well, in case you wish to change